dpl-docs/views/layout.dt

```diff
@@ -113,6 +113,8 @@ html(lang='en-US')
         h1 <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[0 .. $-24]}</code> #{title[$-24 ..$]}
       - else if (hidx > 0 && title[0 .. hidx] != "Module" && info.mod)
         h1 #{title[0 .. hidx]} <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[hidx+1 .. $]}</code>
+      - else if (hidx > 0 && title[0 .. hidx] == "Module" && info.mod)
```

Comment:
I might be a lil pedantic here: `else if (hidx > 0 && title[0 .. hidx + 1] == "Module " && info.mod)`

i.e. space should be part of the matching...better yet, can't we use `title.startsWith("Module ")`? What the limit of stuff usable in these .dt documents?